### ISSUE TITLE
performance issue

### ISSUE THREAD

**Issue by lukysummer** [postBG/DTA.pytorch#10]
Hello, thank you for the paper and code.

Could you possiblly share the **plot of target_ce_loss and target_accuracy for the 20 epochs of training**?

I was training DTA with visda-2017 dataset with **knife, bicycle, and skateboard classes only** with the following command:

**!python main.py --config_path ./configs/resnet101_dta_vat.json --batch_size 50 --experiment_dir logs_DTA_visda_subset --use_vat True**

The only thing i changed from your paper is that I am only using 3 visda classes instead of 12, and batch size is 50 instead of 128 (due to my gpu issue). 

The result is that source_ce_loss continues to decreases and source_accuracy reaches 99% by epoch 2. However, my target_ce_loss continues to increase, and target_accuracy stays around 4~6%.

**Comment by numpee** [comment]
Hi @lukysummer, thanks for your interest in our work.
Sorry for the late reply - we've been busy with another paper submission.

First, I suspect that for the 3 class training, you may need to tune some hyperparameters. Due to the nature of unsupervised training, you may end up with degenerate solutions if the unsupervised loss term (consistency terms, in our case) are too dominant. 

Second, it seems like the accuracy is maybe a little **too** low. Are you sure you're evaluating on just the three classes mentioned? Because even if you have completely random predictions for the target samples, or all your predictions are focused on one class, you should still be getting roughly 33% accuracy. You may want to check this. Let us know if you're still facing any issues.
